packages/core/src/modules/ledger/IndyPool.ts

```diff
@@ -43,6 +49,7 @@ export class IndyPool {
     this.fileSystem = fileSystem
     this.poolConfig = poolConfig
     this.logger = logger
+    this.indyNamespace = createQualifiedIdentifier(this.poolConfig.didIndyNamespace, this.poolConfig.id)
```

Comment:
This doesn't seem right. The namespace is just the didIndyNamespace value right? This will create a full did.

Reply:
ah yeah you're right